rmgpy/cantherm/common.py

```diff
@@ -46,3 +46,171 @@ def checkConformerEnergy(Vlist,path):
         logging.warning('the species corresponding to ' + str(os.path.basename(path)) + ' is different in energy from the lowest energy conformer by ' + "%0.2f" % Vdiff + ' kJ/mol. This can cause significant errors in your computed rate constants. ')
 
 
+def get_element_mass(input_element, isotope=None):
```

Comment:
These large dictionaries should be stored outside of the function declaration, otherwise they get constructed every time you call this function.  It probably doesn't matter that much here, but it's such bad practice I think we should avoid it so future developers don't copy it.

Reply:
Thanks! Fixed and squashed.
The QChem stuff was approved by @amarkpayne. I think this could be merged after tests pass